### PR TITLE
Added a alternative config for rust cargo

### PR DESCRIPTION
**Pull request by Erven2016**
demo-alternative-config-cargo

* added alternative config for rust cargo
* fixed an error in setup configuration (miss symbol `,`)

### CODE REVIEW

File: lua/projectlaunch/alternate_configs/cargo.lua [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+local config_utils = require("projectlaunch.config_utils")
+
+M.name = "Cargo"
+M.runner = "cargo"
+
+M.is_runner_executable = function()
+	if vim.fn.executable(M.runner) == 1 then
+		return true
+	end
+	return false
```
Comment (sheodox) [inline lua/projectlaunch/alternate_configs/cargo.lua:12]:
```suggestion
	return vim.fn.executable(M.runner) == 1
```

File: lua/projectlaunch/alternate_configs/cargo.lua [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+end
+
+M.is_detected = function(project_files)
+	-- matching `src/` and `Cargo.toml`
+	if vim.tbl_contains(project_files, "src") and vim.tbl_contains(project_files, "Cargo.toml") then
+		return true
+	end
+	return false
```
Comment (sheodox) [inline lua/projectlaunch/alternate_configs/cargo.lua:20]:
```suggestion
	return vim.tbl_contains(project_files, "src") and vim.tbl_contains(project_files, "Cargo.toml")
```

**Review by sheodox — COMMENTED** [review]
This is great, thanks! Just a couple clean up suggestions and I'd be ready to merge this!

I had actually done part of this, but never really finished it and never committed any of it as I wasn't familiar enough with rust to know what cargo commands people used, so this is exactly what I needed! Thanks!

**Review by sheodox — APPROVED** [review]

**Comment by sheodox** [comment]
Thanks!

**Comment by Erven2016** [comment]
You're welcome.

**Pull request merged**